src/method-lookup.md

```diff
@@ -6,18 +6,11 @@ file provides an overview of the process. More detailed notes are in
 the code itself, naturally.
 
 One way to think of method lookup is that we convert an expression of
-the form:
+the form `receiver.method(...)` into a more explicit [fully-qualified syntax][]
+(formerly called [UFCS][]):
 
-```rust,ignore
-receiver.method(...)
-```
-
-into a more explicit [UFCS] form:
-
-```rust,ignore
-Trait::method(ADJ(receiver), ...) // for a trait call
-ReceiverType::method(ADJ(receiver), ...) // for an inherent method call
-```
+- `Trait::method(ADJ(receiver), ...)` for a trait call
+- `ReceiverType::method(ADJ(receiver), ...)` for an inherent method call
```

Comment:
Hmm, this doesn't seem any more clear to me? Why did you change it?

Reply:
I think it makes it clearer that it's showing one of two options. The way it was before made it seem like it turned the one line `self.` form into two lines of an explicit form. Also, I think it looks better and is clearer when the comment is turned into proper Markdown text.

Here's a before and after, for comparison:

![image](https://user-images.githubusercontent.com/37223377/114245587-96feb480-9945-11eb-8a2d-d683eaccf9b6.png)

![image](https://user-images.githubusercontent.com/37223377/114245641-b7c70a00-9945-11eb-952a-16c4fc17fda5.png)